timeseries/tests/unittests/models/test_models.py

```diff
@@ -412,6 +413,12 @@ def test_when_custom_metric_passed_to_model_then_model_can_hyperparameter_tune(m
         },
         eval_metric=CustomMetric(),
     )
+    backend = model._get_hpo_backend()
+    if backend is RAY_BACKEND:
+        # Ray has trouble keeping references to the custom metric in the test namespace. We therefore
+        # skip this test.
+        pytest.skip()
```

Comment:
Can we do something like [this](https://auto.gluon.ai/stable/tutorials/tabular/advanced/tabular-custom-metric.html#ensuring-metric-is-serializable) to ensure that Ray works with our implementation of custom metrics?

Reply:
Hmm :/ I don't think so. I don't think the problem is serializability, but rather the fact that during tests Ray doesn't include timeseries.tests.unittests in the pythonpath and find the `CustomMetric` object to unpickle with. If I'm thinking about it, I don't think there's anything more serializable / unserializable about `CustomMetric` than our own metric implementations. I haven't had any trouble so far on this end.